### ISSUE TITLE
Multiple Disallowed IPs

### ISSUE THREAD

**Issue by xDqtreats** [wiresock/WireSockUI#65]
Whenever I go to add multiple disallowed IPs, I run into an issue where my comma isn't separating my IP's.

```
DisallowedIPs = 192.168.1.252,192.168.1.8,192.168.10.104
``` 

This combines the last 2 IP's together. In order for me to make it work, I have to put double commas.

```
DisallowedIPs = 192.168.1.252,192.168.1.8,,192.168.10.104
```

Can anyone else replicate this or tell me my issue?

**Comment by wiresock** [comment]
Could you please test if the issue has gone in WireSock VPN Client v1.2.41?

[WireSock VPN Client x64 v1.2.41.1](https://www.wiresock.net/downloads/wiresock-vpn-client-x64-1.2.41.1.msi)
[WireSock VPN Client ARM64 v1.2.41.1](https://www.wiresock.net/downloads/wiresock-vpn-client-ARM64-1.2.41.1.msi)
[WireSock VPN Client x86 v1.2.41.1](https://www.wiresock.net/downloads/wiresock-vpn-client-x86-1.2.41.1.msi)

**Comment by wiresock** [comment]
Fixed I’m WireSock VPN Client v1.4.5.

**Comment by xDqtreats** [comment]
Thank you. Sorry I didn’t get to try it.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
